bindings/torch/tinycudann/modules.py

```diff
@@ -55,6 +55,79 @@ def backward(ctx, doutput):
 
 		return None, input_grad, weight_grad, None
 
+# NOTE: Jianfei: this is a temporary solution, when only grid has backward_backward and networks do not.
```

Comment:
I would prefer if all modules shared the same `_module_function` and `_module_function_backward` in the interest of avoiding code duplication. Since their interface is equal, this should be possible without overhear, right? (Except that those components not implementing double-bwd will throw an exception.)